chunky/src/java/se/llbit/chunky/world/EmptyChunk.java

```diff
@@ -20,11 +20,13 @@
 import se.llbit.chunky.chunk.ChunkData;
 import se.llbit.chunky.map.IconLayer;
 import se.llbit.chunky.map.MapTile;
+import se.llbit.chunky.ui.ChunkMap;
 import se.llbit.chunky.world.biome.BiomePalette;
 import se.llbit.util.Mutable;
 
 /**
- * Empty or non-existent chunk.
+ * Empty or non-existent chunk in a region that <b>does</b> exist.
+ * In the {@link ChunkMap map view} an {@link EmptyChunk} is represented as white.
```

Comment:
The `{@link ChunkMap map view}` looks invalid to me

Reply:
I believe it is valid, previews as:
![image](https://github.com/chunky-dev/chunky/assets/16853282/b7e8033d-2d93-4972-b193-cb539d4d349c)

Reply:
Didn't know this was possible. Cool!

Reply:
Javadoc task passes locally